optimum/intel/openvino/modeling_decoder.py

```diff
@@ -522,9 +522,11 @@ def forward(
         attention_mask: Optional[torch.LongTensor] = None,
         past_key_values: Optional[Tuple[Tuple[torch.FloatTensor]]] = None,
         position_ids: Optional[torch.LongTensor] = None,
+        token_type_ids: Optional[torch.LongTensor] = None,
         **kwargs,
     ) -> CausalLMOutputWithPast:
         self.compile()
+        kwargs["token_type_ids"] = token_type_ids
```

Comment:
can you add a comment so that we know why this was added, something like :
```suggestion
        # added as model.generate validates model inputs based on forward signature
        kwargs["token_type_ids"] = token_type_ids
```